### PR TITLE
scripts: build and deploy PDF pages for translations

### PR DESCRIPTION
**Pull request by kbdharun**
Superseeds #10846 

## Fixes

- Updated the failing deploy script.

### CODE REVIEW

**Comment by kbdharun** [comment]
@acuteenvy If possible, can you check out this PR?

Basically, we have tested and verified building PDFs, but the `deploy.sh` script seems to fail during execution (see https://github.com/tldr-pages/tldr/pull/10846#issuecomment-1760891461). Feel free to fix it in the branch.

**Comment by acuteenvy** [comment]
> cp: cannot stat '/home/runner/work/tldr/tldr/scripts/pdf/tldr-book*.pdf': No such file or directory

Shell wildcards are not expanded in single quotes. The shell doesn't expand `*`, `cp` tries to copy `tldr-book*.pdf` - the file doesn't exist, and so it fails.

**Review by acuteenvy — APPROVED** [review]
And it seems you have fixed it yourself.

File: scripts/pdf/render.py [hunk 1]
```diff
@@ -89,6 +91,12 @@ def main(loc, colorscheme):
         default="basic",
         help="Color scheme of the PDF",
     )
+    parser.add_argument(
+        "-o",
+        "--output",
+        default="tldr-book.pdf",
+        help="Custom filename for the output PDF (default is 'tldr-pages.pdf')",
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply scripts/pdf/render.py:98]: Nice catch, fixed it.

```suggestion
        help="Custom filename for the output PDF (default is 'tldr-book.pdf')",
```

**Comment by kbdharun** [comment]
Now, we wait to see if the deploy succeeds or else I will revert the change like last time.

**Comment by kbdharun** [comment]
**Update:** The deploy works and the PDF for translations has been deployed successfully.

Now, we need to document all the PDF files per language maybe as a table in either wiki or in the repo itself.

**Review by timscharfenort8 — COMMENTED** [review]
Good work

**Pull request merged**